.github/workflows/black.yml

```
@@ -0,0 +1,41 @@
+    branches: [ main, master ]
+
+# The actual jobs that will run
+jobs:
+  formatting:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed
+    steps:
+    # Checkout the repository code
+    - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)